### DISCUSSION TITLE
Error in Creating Arrow Files

### DISCUSSION THREAD

**Discussion by jaesoonpark** [GreenleafLab/ArchR#629] [Questions / Documentation] [answered]
[ArchR-createArrows-c5b539cb0b55-Date-2021-03-17_Time-13-55-13.log](https://github.com/GreenleafLab/ArchR/files/6153868/ArchR-createArrows-c5b539cb0b55-Date-2021-03-17_Time-13-55-13.log)

Thank you for providing the useful tool.
I am at the first stage of the pipeline and got error in converting fragment (bed) file into arrow file. 
Is there a specific format of fragment file that must be followed? 
Currently, my bed file includes
(1) Chromosome
(2) Start
(3) End
(4) Barcode 
(5) Number 
(6) Blank (with all the data written as .)

Please let me know if I can solve this issue.

Thanks.

**Comment by rcorces** [comment]
Thanks for using ArchR! One note - questions and feature requests should be submitted through the Discussions forum (https://github.com/GreenleafLab/ArchR/discussions). I will migrate this issue there now but please only use Issue reports for actual bugs in the software.

**Comment by rcorces** [comment, marked as answer]
Fragment files are meant to follow the format from 10x Genomics which is described here:
https://support.10xgenomics.com/single-cell-atac/software/pipelines/latest/output/fragments

Thanks for pointing out that this was missing from the documentation. I have made a note to add it whenever we push the next update to documentation.
